src/components/transport_manager/include/transport_manager/transport_adapter/transport_adapter_impl.h

```diff
@@ -400,6 +400,15 @@ class TransportAdapterImpl : public TransportAdapter,
    */
   virtual std::string GetConnectionType() const;
 
+  /**
+   * @brief Check if the state of the specified connection is FINILIZING.
+   *
+   * @param device_handle Device unique identifier.
+   * @param app_handle Handle of application.
+   */
+  virtual bool IsConnectionFinished(const DeviceUID& device_handle,
```

Comment:
@vlantonov Please add OVERRIDE macro to all virtual methods. We have our own macro (it's not c++11 yet)